src/aws/generateRoleParams.js

```diff
@@ -1,7 +1,7 @@
 const generateRolePolicy = require("./generateRolePolicy");
 
 const generateRoleParams = (roleName) => {
-  const service = roleName.startsWith("lambda") ? "lambda" : "states";
+  const service = roleName.includes("lambda") ? "lambda" : "states";
```

Comment:
Looks great! It may be worth noting that this could also be implemented with the `endsWith` method, but the end result is the same.